Cargo.toml

```diff
@@ -46,6 +46,8 @@ avformat-sys = { version = "0.1", path = "crates/avformat-sys/", optional = true
 rayon = "1.0"
 bincode = "1.1"
 arrayvec = "0.4.10"
+image = "0.21.2"
+byteorder = "1.3.2"
```

Comment:
For this dep, we ended up just using bitstream-io everywhere, which is basically a superset of what byteorder offers.